mw/auth/jwt_test.go

```diff
@@ -0,0 +1,83 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	TEST_SECRET = "some-secret-123"
+)
+
+func TestGetTenantID(t *testing.T) {
+	var tenantIDTests = []struct {
+		claims   jwt.MapClaims
+		field    string
+		expected string
+		err      error
+	}{
+		{
+			claims: jwt.MapClaims{
+				"some-field": "id-abc-123",
+			},
+			field:    "some-field",
+			expected: "id-abc-123",
+			err:      nil,
+		},
+		{
+			claims: jwt.MapClaims{
+				"some-field": "id-abc-123",
+			},
+			field:    "some-other-field",
+			expected: "",
+			err:      errMissingField,
+		},
+		{
+			claims:   jwt.MapClaims{},
+			field:    "some-field",
+			expected: "",
+			err:      errMissingToken,
+		},
+	}
+	for _, test := range tenantIDTests {
+		ctx := context.Background()
+		if len(test.claims) != 0 {
+			token := makeToken(test.claims, t)
+			ctx, _ = contextWithToken(token)
```

Comment:
is the `_` an `error`? If so, you should check it and call a `t.Fatalf` if it exists.

Reply:
Yeah, that returns an error. Sloppy coder strikes again.